doc/api/tls.md

```diff
@@ -821,6 +821,28 @@ The `callback` function, if specified, will be added as a listener for the
 
 `tls.connect()` returns a [`tls.TLSSocket`][] object.
 
+To upgrade an existing instance of `net.Socket` to a 
+`tls.TLSSocket`, pass it to `tls.connect()` as 
+the socket option:
+
+```js
+const { Socket } = require('net');
+const tls = require('tls');
+const socket = new Socket();
+const secureSock = tls.connect({ socket }, () => {
+  console.log('The TLS socket has been connected.');
```

Comment:
This must have slipped through, earlier commits had the actual `connect` code in them.